from_cpython/Include/object.h

```diff
@@ -479,6 +479,7 @@ typedef struct _heaptypeobject {
                                       see add_operators() in typeobject.c . */
     PyBufferProcs as_buffer;
     PyObject *ht_name, *ht_slots;
+    Py_ssize_t nslots;
```

Comment:
CPython uses ob_size for this field; could we do the same so that we don't have to add a new field?